extra_data/tests/conftest.py

```diff
@@ -8,7 +8,7 @@
 from . import make_examples
 
 
-@pytest.fixture(scope='session', params=['0.5', '1.0'])
+@pytest.fixture(scope='session', params=['0.5', '1.0', '1.2'])
```

Comment:
I'm not super keen on running a whole load of tests again with only a trivial difference, but we can live with that.

If we are going to mock format version 1.2, though, we should do it properly, which means adding the `INDEX/origin` dataset to the mocked data. I think it's fine to fill it all with `-1` for now.

https://github.com/European-XFEL/EXtra-data/blob/dc66ddab58d6d7d0a05a318c5fdb0c76a07e5114/extra_data/tests/mockdata/base.py#L189-L198

Reply:
Alrighty, added in 3cb8ded.